src/frontend/sharedComponents/MediaScrollView/PhotoThumbnail.tsx

```diff
@@ -56,6 +79,46 @@ export const PhotoThumbnail: FC<PhotoThumbnailProps> = props => {
   );
 };
 
+const SavedPhotoThumbnail = ({photo}: {photo: SavedPhoto}) => {
+  const image = useAttachmentUrlQuery(photo, 'thumbnail');
+
+  return (
+    <PhotoThumbnailImage
+      isLoading={image.isPending}
+      error={image.error}
+      uri={image.data?.url}
+    />
+  );
+};
+
+const UnsavedPhotoThumbnail = ({photo}: {photo: DraftPhoto}) => {
+  return (
+    <PhotoThumbnailImage
+      isLoading={photo.type === 'unprocessed' && !('error' in photo)}
+      error={'error' in photo ? photo.error : undefined}
+      uri={'thumbnailUri' in photo ? photo.thumbnailUri : undefined}
+    />
+  );
+};
+
+type PhotoThumbnailContextProps = {
+  onPress: () => any;
```

Comment:
adding the optional here makes more sense in the context of my suggestion for cleaning up the `MediaScrollView` component:

```suggestion
  onPress?: () => void;
```